titus_isolate/config/restart_property_watcher.py

```diff
@@ -11,35 +14,28 @@ class RestartPropertyWatcher:
 
     def __init__(
             self,
-            config_manager,
-            exit_handler,
-            properties,
-            detection_interval=PROPERTY_CHANGE_DETECTION_INTERVAL_SEC):
+            config_manager: ConfigManager,
+            exit_handler: ExitHandler,
+            properties: List[str],
+            detection_interval: int = PROPERTY_CHANGE_DETECTION_INTERVAL_SEC):
 
         self.__config_manager = config_manager
         self.__exit_handler = exit_handler
         self.__properties = properties
 
-        self.__original_properties = {}
-        for p in properties:
-            self.__original_properties[p] = config_manager.get_str(p)
-
-        self.__original_primary_allocator_name =\
-            get_fallback_allocator(config_manager).get_primary_allocator().__class__.__name__
-
         log.info("Starting watching for changes to properties: {}".format(properties))
-        for k, v in self.__original_properties.items():
-            log.info("{}: {}".format(k, v))
+        for p in properties:
+            v = config_manager.get_cached_str(p)
+            log.info("{}: {}".format(p, v))
 
         schedule.every(detection_interval).seconds.do(self.detect_changes)
 
     def detect_changes(self):
         self.__detect_property_changes()
-        self.__detect_ab_changes()
 
     def __detect_property_changes(self):
```

Comment:
Is this function just called periodically somewhere?

Reply:
Look up a few lines
`schedule.every`...

Reply:
Ah, derp. Neat.